td-client.gemspec

```diff
@@ -26,6 +26,9 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'simplecov', '>= 0.5.4'
   gem.add_development_dependency 'rake'
   gem.add_development_dependency 'yard'
+  if RUBY_VERSION >= '3.0'
```

Comment:
This file is interpreted by a particular Ruby version on `gem push`, which means the dependency definition will be changed what Ruby version you'll use for `gem push`. 

You should simply declare the dependency regardless of the Ruby version if you really want to make sure it's specified as a development dependency when you use the gem from Ruby 3.0. Development dependencies on released packages probably don't really matter though.

Reply:
Ah I see. I have wrong understanding of RUBY_VERSION in gemspec. Thanks for catching it!